modules/installation-user-defined-tags-requirements-azure.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// * installing/installing_azure/installing-azure-customizations.adoc
+:_mod-docs-content-type: REFERENCE
+[id="installation-user-defined-tags-requirements-azure_{context}"]
+= User-defined tags requirements
+
+The user-defined tags have the following requirements:
+
+* A tag key can have a maximum of 128 characters.
+
+* A tag key must begin with a letter, end with a letter, number or underscore, and can contain only letters, numbers, underscores(`_`), periods(`.`), and hyphens(`-`).
+
+* Tag keys are case-insensitive.
+
+* Tag keys cannot be `name`. It cannot have prefixes such as `kubernetes.io`, `openshift.io`, `microsoft`, `azure`, and `windows`.
```

Comment:
```suggestion
* A tag key must not be called `name`. 

* A tag key must not use the following prefixes:

** `azure`
** `kubernetes.io`
** `microsoft`
** `openshift.io`
** `windows`
```

Reply:
Thanks a lot for pointing this out! I think I totally missed reviewing this piece after pulling it out in to this module.